### DISCUSSION TITLE
Renault Trafic III

### DISCUSSION THREAD

**Discussion by molnar25** [cedricp/ddt4all#1023] [Q&A]
Hi I have a renault Trafic III 2.0blue dci 2023, is there no database for this model for android?

**Comment by AndyIreland** [comment]
I have traffic III listed in my DB : (but this is Windows DDT4ALL DB) 
![image](https://github.com/cedricp/ddt4all/assets/49172217/9d341f3f-15c0-4b0d-8e72-a8c243f01559)

**Reply by molnar25** [reply]
Can you show me the engine control unit?

**Reply by monalos** [reply]
Click on the magnifying glass icon at the top right in the image, it will automatically find the ecu's. @molnar25

**Reply by molnar25** [reply]
Where should I click on the magnifying glass icon in the android application?

**Comment by AndyIreland** [comment]
when you have selected your car with the magnyfying glass, and chose your ECU , then click on 'auto identify' button:
![430197034_1184867239564071_5860116629737314176_n](https://github.com/cedricp/ddt4all/assets/49172217/568b8214-ff18-4906-beca-0345b38ac43f)

**Reply by molnar25** [reply]
cannot find the Engine ECU. power steering only.

**Reply by AndyIreland** [reply]
could be incompatible OBD2 ELM327 adaptor maybe? has it got required PIC18F25K80 chip in it?

**Reply by molnar25** [reply]
I think so, because the Car scanner application reads values from the engine ecu..
